module/commands/menu.py

```diff
@@ -0,0 +1,56 @@
+""" /menu command """
+from module.data.constants import DB_PATH
+from module.data.user_settings import UserSettings
+from telegram import InlineKeyboardButton, InlineKeyboardMarkup, Update
+from telegram.ext import CallbackContext, CallbackQueryHandler, CommandHandler, ContextTypes
+
+def menu(update: Update, context: CallbackContext) -> None:
+    update.message.reply_text("Imposta quando ricevere il menù con /menu_settings")
+    return
+
+def menu_settings(update: Update, context: CallbackContext) -> None:
+    menu_set = UserSettings()
+
+#    keyboard = [
+#        GIORNI_PRANZO_CENA,
+#        [
+#            InlineKeyboardButton("Lunedì", callback_data="0"),
+#            InlineKeyboardButton("❌", callback_data="1"),
```

Comment:
we could put `"❌"` in a constant variable like `CROSS`

Reply:
ok, perfect